### PR TITLE
replace lang3 in api

### PR DESCRIPTION
**Pull request by lin-mt**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

#3102

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [x] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Comment by lin-mt** [comment]
@KomachiSion Hello，可以帮忙看下为什么 Travis CI 失败了吗。

File: api/src/main/java/com/alibaba/nacos/api/naming/pojo/healthcheck/impl/Http.java [hunk 1]
```diff
@@ -32,87 +32,88 @@
...
+    
+    public void setHeaders(final String headers) {
         this.headers = headers;
     }
-
+    
     @JsonIgnore
     public Map<String, String> getCustomHeaders() {
-        if (StringUtils.isBlank(headers)) {
+        if (StringUtils.isBlank(this.headers)) {
```
Comment (KomachiSion) [inline api/src/main/java/com/alibaba/nacos/api/naming/pojo/healthcheck/impl/Http.java:74]: No need use `this` for use except in constructor, set, get.

File: api/src/main/java/com/alibaba/nacos/api/naming/pojo/healthcheck/impl/Http.java [hunk 2]
```diff
@@ -32,87 +32,88 @@
...
             return Collections.emptyMap();
         }
-        Map<String, String> headerMap = new HashMap<String, String>(16);
-        for (String s : headers.split(Constants.NAMING_HTTP_HEADER_SPILIER)) {
-            String[] splits = s.split(":");
+        final Map<String, String> headerMap = new HashMap<String, String>(16);
+        for (final String s : this.headers.split(Constants.NAMING_HTTP_HEADER_SPILIER)) {
```
Comment (KomachiSion) [inline api/src/main/java/com/alibaba/nacos/api/naming/pojo/healthcheck/impl/Http.java:78]: No need use this for use except in constructor, set, get.

File: api/src/main/java/com/alibaba/nacos/api/naming/pojo/healthcheck/impl/Http.java [hunk 3]
```diff
@@ -32,87 +32,88 @@
...
             headerMap.put(StringUtils.trim(splits[0]), StringUtils.trim(splits[1]));
         }
         return headerMap;
     }
-
+    
     @Override
     public int hashCode() {
-        return Objects.hashCode(path, headers, expectedResponseCode);
+        return Objects.hashCode(this.path, this.headers, this.expectedResponseCode);
```
Comment (KomachiSion) [inline api/src/main/java/com/alibaba/nacos/api/naming/pojo/healthcheck/impl/Http.java:90]: No need use this for use except in constructor, set, get.

File: api/src/main/java/com/alibaba/nacos/api/naming/pojo/healthcheck/impl/Http.java [hunk 4]
```diff
@@ -32,87 +32,88 @@
...
             return false;
         }
-
-        Http other = (Http) obj;
-
-        if (!StringUtils.equals(type, other.getType())) {
+        
+        final Http other = (Http) obj;
+        
+        if (!StringUtils.equals(this.type, other.getType())) {
```
Comment (KomachiSion) [inline api/src/main/java/com/alibaba/nacos/api/naming/pojo/healthcheck/impl/Http.java:101]: No need use this for use except in constructor, set, get.

File: api/src/main/java/com/alibaba/nacos/api/naming/pojo/healthcheck/impl/Http.java [hunk 5]
```diff
@@ -32,87 +32,88 @@
...
+        
+        if (!StringUtils.equals(this.path, other.getPath())) {
             return false;
         }
-        if (!StringUtils.equals(headers, other.getHeaders())) {
+        if (!StringUtils.equals(this.headers, other.getHeaders())) {
             return false;
         }
-        return expectedResponseCode == other.getExpectedResponseCode();
+        return this.expectedResponseCode == other.getExpectedResponseCode();
```
Comment (KomachiSion) [inline api/src/main/java/com/alibaba/nacos/api/naming/pojo/healthcheck/impl/Http.java:111]: No need use this for use except in constructor, set, get.

File: api/src/main/java/com/alibaba/nacos/api/exception/NacosException.java [hunk 6]
```diff
@@ -13,136 +13,145 @@
...
+     * bad gateway（路由异常，如nginx后面的Server挂掉）.
+     */
+    public static final int BAD_GATEWAY = 502;
+    
+    /**
+     * over threshold（超过server端的限流阈值）.
+     */
+    public static final int OVER_THRESHOLD = 503;
+    
+    public static final int RESOURCE_NOT_FOUND = -404;
```
Comment (yanlinly) [inline api/src/main/java/com/alibaba/nacos/api/exception/NacosException.java:156]: 这个码是什么意思？

Comment (lin-mt) [reply api/src/main/java/com/alibaba/nacos/api/exception/NacosException.java:156]: -404 吗，这个之前就已经存在了，是 @nkorange 添加的

Comment (lin-mt) [reply api/src/main/java/com/alibaba/nacos/api/exception/NacosException.java:156]: > 这个码是什么意思？

-404 吗，这个之前就已经存在了，是 @nkorange 添加的

File: api/src/main/java/com/alibaba/nacos/api/naming/pojo/healthcheck/impl/Http.java (hunk 5 above)
Comment (lin-mt) [reply api/src/main/java/com/alibaba/nacos/api/naming/pojo/healthcheck/impl/Http.java]: 去掉 this 了哈

**Pull request merged**